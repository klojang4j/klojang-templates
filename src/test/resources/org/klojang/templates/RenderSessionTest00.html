<html>
<body>
	<table>
		<thead>
			<tr>
				<th>Company Name</th>
				<th>PO Box</th>
				<th>Established</th>
				<th>Director</th>
				<th>Department</th>
			</tr>
		</thead>
		<tbody>
		~%%begin:company%
			<tr onclick="showPopup('~%js:name%');">
				<td>~%name%</td>
				<td>~%poBox%</td>
				<td>~%established%</td>
				<td>~%director%</td>
				<td>~%%include:departments:include.departments.html%</td>
			</tr>	
		~%%end:company%
		</tbody>
	</table>
</body>
</html>