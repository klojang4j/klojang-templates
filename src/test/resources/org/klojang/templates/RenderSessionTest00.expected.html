<html>
<body>
	<table>
		<thead>
			<tr>
				<th>Company Name</th>
				<th>PO Box</th>
				<th>Established</th>
				<th>Director</th>
				<th>Department</th>
			</tr>
		</thead>
		<tbody>
		
			<tr onclick="showPopup('Shell');">
				<td>Shell</td>
				<td>AB12345</td>
				<td>1932-04-04</td>
				<td></td>
				<td><table>
<thead>
	<tr>
		<th>Name</th>
		<th>Manager</th>
		<th>Employees</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>HR</td>
		<td>Kristina Aguilera</td>
		<table>
<thead>
	<tr>
		<th>Name</th>
		<th>Ages</th>
		<th>Roles</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>John Travolta</td>
		<td>M</td>
		<td>1963-03-03</td>
	</tr>
</tbody>
</table><table>
<thead>
	<tr>
		<th>Name</th>
		<th>Ages</th>
		<th>Roles</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>Tony Chocolonely</td>
		<td>M</td>
		<td>1972-04-04</td>
	</tr>
</tbody>
</table><table>
<thead>
	<tr>
		<th>Name</th>
		<th>Ages</th>
		<th>Roles</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>Queen Elisabeth</td>
		<td>F</td>
		<td>1922-01-02</td>
	</tr>
</tbody>
</table>
	</tr>
</tbody>
</table><table>
<thead>
	<tr>
		<th>Name</th>
		<th>Manager</th>
		<th>Employees</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>ICT</td>
		<td>Woody Harrelson</td>
		<table>
<thead>
	<tr>
		<th>Name</th>
		<th>Ages</th>
		<th>Roles</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>George Clooney</td>
		<td>M</td>
		<td>1962-01-02</td>
	</tr>
</tbody>
</table><table>
<thead>
	<tr>
		<th>Name</th>
		<th>Ages</th>
		<th>Roles</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>Joe Biden</td>
		<td>M</td>
		<td>1942-08-02</td>
	</tr>
</tbody>
</table>
	</tr>
</tbody>
</table></td>
			</tr>	
		
		</tbody>
	</table>
</body>
</html>